src/Api.js

```diff
@@ -183,6 +183,8 @@ class Api {
 		revId = token[ 2 ];
 		revision = this.results.revisions[ revId ];
 		username = revision[ 3 ];
+		isIP = username.slice( 0, 2 ) === '0|';
```

Comment:
What's the format that we're getting from the API that starts with this? Can you quickly add a comment above? it looks weird to have a `'0|` as an indication of an IP address unless it's how the API specifically gives it. Does the API give us `username|detail` and then username is 0 if it's not logged in?

Just add a quick comment right above that to clarify for the future.

Reply:
Yup, this is how WikiWho differentiates accounts from IPs. This code was refactored from https://github.com/wikimedia/WhoWroteThat/blob/997016b08ed367e0a9bd47865bcb5989b8de99ee/src/RevisionPopupWidget.js#L46-L47 I should have added a comment there.